src/index.html

```diff
@@ -11,7 +11,164 @@
+      <fieldset class="field">
+        <legend>Personal information</legend>
+
+        <label class="surname personal-inf">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="name personal-inf">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="age personal-inf">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="personal-inf">
+          Full date of birth:
+          <input type="date" name="date">
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>Registration</legend>
+
+        <label class="personal-inf">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="personal-inf">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cat"
+            value="y"
+          >
+          Yes
+          <input
+            type="radio"
+            name="cat"
+            value="n"
+          >
+          No
+        </label>
+
+        <label class="personal-inf">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+
+        <label class="personal-inf">
+          What time do you go to bad?
+          <input type="time" name="time">
+        </label>
+
+        <label class="personal-inf">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            id="favcar"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label>
+          How do you rate our work?
+          <input
+            type="range"
+            name="range"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>Additional info:</legend>
+
+        <label class="personal-inf">
+          Comments:
```

Comment:
checklist p.8 - Check the textarea behaviour - when you click on the textarea, the coursor should appear on the beginning of the input.